src/Compilers/Core/Portable/Diagnostic/Diagnostic.DiagnosticWithProgrammaticSuppression.cs

```diff
@@ -38,10 +40,10 @@ public override string Id
                 get { return Descriptor.Id; }
             }
 
-            public override string GetMessage(IFormatProvider formatProvider = null)
+            public override string? GetMessage(IFormatProvider? formatProvider = null)
                 => _originalUnsuppressedDiagnostic.GetMessage(formatProvider);
 
-            internal override IReadOnlyList<object> Arguments
+            internal override IReadOnlyList<object>? Arguments
```

Comment:
💭 Many times arguments to a format method allow null. You might check to see if the following is appropriate:

```suggestion
            internal override IReadOnlyList<object?>? Arguments
```

Reply:
So far I haven't seen the need for this. I want to try and avoid the need for extra nulls until my hand gets forced. 

---
In reply to: [356884157](https://github.com/dotnet/roslyn/pull/40268#discussion_r356884157) [](ancestors = 356884157)